armi/reactor/reactors.py

```diff
@@ -158,7 +159,15 @@ def factory(cs, bp, geom: Optional[SystemLayoutInput] = None) -> Reactor:
 
     runLog.debug("Reactor: {}".format(r))
 
-    r.sort()
+    # return a Reactor object
+    if cs[CONF_SORT_REACTOR]:
+        r.sort()
+    else:
+        runLog.warning(
+            "DeprecationWarning: This Reactor is not being sorted on DB load. "
+            f"Due to the setting {CONF_SORT_REACTOR}, this Reactor is unsorted. "
+            "But this feature is temporary and will be removed by 2024."
+        )
```

Comment:
A copy paste error here I think. Should be "DeprecationWarning: This reactor is not being sorted on reactor construction due to the setting..."

Reply:
Ah, right you are. 1 sec.

Reply:
Fixed! Thanks!